x-pack/winlogbeat/module/security/_meta/fields.yml

```diff
@@ -0,0 +1,41 @@
+- key: security
+  title: "Security module"
+  description: >
+    These are the event fields specific to the module for the Security log.
+  release: beta
+  fields:
+
+    - name: winlog.logon
+      type: group
+      description: Windows status code returned for the DNS query.
```

Comment:
Is this description correct? What has a logon to do with DNS queries?

Reply:
Copy/paste issue. I'm glad you read the descriptions. 👍